client/post-editor/editor-diff-viewer/style.scss

```diff
@@ -32,15 +32,16 @@
 	white-space: pre-wrap;
 	padding: 0;
 	background: transparent;
+	overflow-y: hidden;
 }
 
 .editor-diff-viewer__additions {
-	background: lighten( $blue-wordpress, 58% );
+	background: lighten($blue-wordpress, 58%);
```

Comment:
The two spacing changes were added by prettier on commit. 🤔

Reply:
I've noticed prettier doing the same to CSS. I'm not sure if this is the desired behavior or if we need to update [our fork](https://github.com/Automattic/calypso-prettier).